Library/Formula/aurora-scheduler.rb

```diff
@@ -0,0 +1,18 @@
+class AuroraScheduler < Formula
+  desc "Apache Aurora Scheduler Client"
+  homepage "http://aurora.apache.org/"
+  url "http://www.us.apache.org/dist/aurora/0.9.0/apache-aurora-0.9.0.tar.gz"
+  version "0.9.0"
+  sha256 "16040866f3a799226452b1541892eb80ed3c61f47c33f1ccb0687fb5cf82767c"
+
+  depends_on "python" => :build
```

Comment:
Should be either `depends_on :python => :build` or `depends_on :python => :build if MacOS.version <= :snow_leopard`, I believe. @tdsmith